src/helpers/openai.ts

```diff
@@ -0,0 +1,45 @@
+import { readSecret } from "./secrets";
+
+const { OpenAIClient, AzureKeyCredential } = require("@azure/openai");
```

Comment:
Let's use import instead of require. That's better as it's more modern and is part of ES modules.

Reply:
Updated 👍